### PR TITLE
create green_list database with tables

### PR DESCRIPTION
**Pull request by aransagarcia**

### CODE REVIEW

**Comment by joseph-p-pasaoa** [comment]
Will not merge this pull request until other pull request is approved or rejected:
[Pull Req: added number limits to varchars, fix typos](https://github.com/joseph-p-pasaoa/greenlistRegistry_back__Web/pull/1)
Then will merge these in order

**Review by KathyPuma — APPROVED** [review]

**Pull request merged**
